tests/ui/perfherder/modal_perf_tags_test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
We should also reorganize the structure of some test files.
E.g. create a dedicated test dir for **Alerts view** & in it add this module **+** the current `alerts_tests.jsx` module.